From 5c2cbfccf9aafb547b0b30914c4056abd25942a4 Mon Sep 17 00:00:00 2001
From: Allan Sandfeld Jensen <allan.jensen@qt.io>
Date: Fri, 6 Jan 2017 15:30:47 +0100
Subject: [PATCH] Avoid overriding new and delete

As a minimum this produces valgrind warnings of mismatched new and
delete. No runtime issues have been observed outside of valgrind though.

Change-Id: I5ebb6b380f25f117434633e55dd7fdf04260f0bc
Reviewed-by: David Faure <david.faure@kdab.com>
---
 src/core/config/linux.pri | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/core/config/linux.pri b/src/core/config/linux.pri
index 5941799..b91e795 100644
--- a/src/core/config/linux.pri
+++ b/src/core/config/linux.pri
@@ -21,7 +21,9 @@ GYP_CONFIG += \
     use_gnome_keyring=0 \
     use_kerberos=0 \
     use_pango=0 \
-    use_openssl=1
+    use_openssl=1 \
+    use_allocator=none \
+    use_experimental_allocator_shim=0
 
 use?(nss) {
     GYP_CONFIG += \
